libcudacxx/include/cuda/std/detail/libcxx/include/cmath

```diff
@@ -973,7 +973,7 @@ _LIBCUDACXX_INLINE_VISIBILITY constexpr _Fp __lerp(_Fp __a, _Fp __b, _Fp __t) no
     return __b;
   }
   const _Fp __x = __a + __t * (__b - __a);
-  if (__t > 1 == __b > __a)
+  if ((__t > 1) == (__b > __a))
```

Comment:
o.O

Reply:
I somehow landed in that file, saw this, got confused, and hit the autofix button because there was a yellow squiggle underneath that expression, and then forgot about it. Sorry about that, the code before was fine too. I will revert.

Reply:
Seems like gcc really wants those braces, so lets just add them

```
/home/coder/cccl/libcudacxx/lib/cmake/libcudacxx/../../../include/cuda/std/detail/libcxx/include/cmath:976:11: error: suggest parentheses around comparison in operand of '==' [-Werror=parentheses]
    976 |   if (__t > 1 == __b > __a)
```